presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergLogicalPlanner.java

```diff
@@ -0,0 +1,437 @@
+/*
```

Comment:
Quick question: Is this test based on the `TestHiveLogicalPlanner.java`?

Reply:
Yes, I did copy it from `TestHiveLogicalPlanner`